config.rb

```diff
@@ -84,6 +84,13 @@
   deploy.method = :git
   deploy.build_before = true
   deploy.branch = 'master'
+
+  case ENV['TARGET'].to_s.downcase
+  when 'production'
+    deploy.remote = 'origin'
+  else
+    deploy.remote = 'git@github.com:RSpec-Staging/rspec-staging.github.io.git'
```

Comment:
Note this seems to be necessary because it didn't like me specifying the remote name, but this has the added advantage of not needing to know about the remote.